packages/styles/icon-button.css

```diff
@@ -1,46 +1,41 @@
 :root {
+  --icon-button-content-color-light-disabled: #ccc;
+  --icon-button-background-color-light: #dadada;
+  --icon-button-background-color-light-active: #c2c2c2;
+  --icon-button-outline-color-light: var(--icon-button-background-color-light);
+
+  --icon-button-content-color-dark-disabled: #fff 40%;
+  --icon-button-background-color-dark: var(--accent-medium);
+  --icon-button-background-color-dark-disabled: var(--accent-medium);
+  --icon-button-background-color-dark-active: var(--accent-dark);
+  --icon-button-outline-color-dark: var(--icon-button-background-color-dark);
```

Comment:
I can't find these variables used anywhere. We should [mark these as deprecated](https://github.com/dequelabs/cauldron/blob/develop/CONTRIBUTING.md#styles-1), and add a ticket to remove these in the future as a breaking change.

Since these should also not be used, they should be positioned back towards the bottom of `:root`.